subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationSpec.groovy

```diff
@@ -765,6 +764,35 @@ class DefaultConfigurationSpec extends Specification {
         assert copiedConfiguration.extendsFrom.empty
     }
 
+    def "Calls resolve actions and closures from original configuration when copied configuration is resolved"() {
+        Action<ResolvableDependencies> beforeResolveAction = Mock()
+        Action<ResolvableDependencies> afterResolveAction = Mock()
+        def config = conf("conf")
+        def beforeResolveCalled
```

Comment:
We should probably initialize both fields with `false` because it's not strongly typed and therefore would have a `null` default value.